patientsearch/api.py

```diff
@@ -70,7 +70,10 @@ def main():
     ## issue with path resolution after build
     return send_from_directory(
         #todo: remove templates directory reference; index.html isn't a jinja template
-        safe_join(current_app.static_folder, 'templates'),
+        safe_join(
+            current_app.config.get("STATIC_DIR") or current_app.static_folder,
+            'templates'
```

Comment:
Wouldn't you expect that IF `STATIC_DIR` is defined, it'll be in `current_app.static_folder` as per the `create_app()` code?

Reply:
You're right, it doesn't matter.  Merging.